test/test_pebble.py

```diff
@@ -479,7 +479,7 @@ def test_dict(self):
         s = pebble.Service('Name 2', d)
         self.assertEqual(s.name, 'Name 2')
         self.assertEqual(s.description, 'The lazy quick brown')
-        self.assertEqual(s.default, 'Dee Fault')
+        self.assertEqual(s.startup, 'Start Up')
```

Comment:
so do we validate the content of the fields, or we assume that pebble will be checking that enabled/disabled vs arbitrary content?

Reply:
Yeah, for these fields we don't validate -- we assume that Pebble has already validated the configuration.

Reply:
@benhoyt @jameinel we can look at doing more validation on these values in the python classes, but for now they are validated server side which should be sufficient.